sdk/communication/Azure.Communication.Administration/tests/PhoneNumberAdministrationClient/PhoneNumberAdministrationClientLiveTests.cs

```diff
@@ -58,6 +58,17 @@ public async Task GetAllPhoneNumbers()
             Assert.IsNotNull(numbers);
         }
 
+        [Test]
+        public async Task GetAllReservations()
+        {
+            var client = CreateClient();
+
+            var reservationsPagable = client.GetAllReservationsAsync();
+            var reservations = await reservationsPagable.ToEnumerableAsync();
+
+            Assert.IsNotEmpty(reservations);
```

Comment:
Can we guarantee that this is not empty on a fresh created resource and where we can't guarantee the order in which the tests execute?

Consider `IsNotNull` if we're not sure.

Reply:
I have added CreateReservation step to the test so there will be at least one reservation